src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -116,13 +121,17 @@ public ConfigurationResponse createConfiguration(String vmName, String image, St
     }
 
     public DeploymentResponse deployVM(String vmName) throws IOException {
-        return this.deployVM(vmName, null);
+        return this.deployVM(vmName, null, null);
```

Comment:
`nit`: You can leave it as it was. The second overload will call the new one with the correct scheduler